.travis.yml

```diff
@@ -4,7 +4,9 @@ language: ruby
 # shipped with Ruby 2.5.
 dist: bionic
 cache: bundler
-script: bundle exec middleman build
+script:
+  - bundle exec middleman build
+  - npm test
```

Comment:
I think I'd prefer to avoid the indirection and just run `bundle exec rake` from here, and not add it to `package.json` – either way it's a bit weird as we're mixing ruby and node tooling, but wrapping the ruby tooling in node tooling seems to add to the confusion rather than simplifying things. What do you think?

Reply:
Yes that makes sense to me too 👍